app/delivery/send_to_providers.py

```diff
@@ -58,11 +58,16 @@ def send_sms_to_provider(notification):
 
         else:
             try:
+                recipient = notification.to
+                is_international = notification.international
+                notification_id = notification.id
+                reply_to_text = notification.reply_to_text
+                db.session.commit()
```

Comment:
I’m sure these lines are unlikely to raise an exception, but if they ever do it will change the provider priority because they’re within the `try` block. That doesn’t feel right to me.